site/public/js/server.js

```diff
@@ -107,19 +107,25 @@ function changeDiffView(div_name, gradeable_id, who_id, version, index, autochec
 }
 
 function loadTestcaseOutput(div_name, gradeable_id, who_id, index, version = ''){
-    orig_div_name = div_name
+    let orig_div_name = div_name;
```

Comment:
It may be irrelevant to this PR, but I wonder if we are supposed to use ES6 keywords.

In `server.js`, I only found 1 occurrence of `const` and 4 occurrences of `let`. Does that mean it is better to avoid using ES6 because of [browser compatibility](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let#Browser_compatibility), or it is completely fine to use them in this repo (considering that the scope of `let` is often clearer than `var`)?

Reply:
I see. Thank you for your response!